src/customers/components/CustomerAddressChoiceCard/styles.ts

```diff
@@ -3,12 +3,14 @@ import { makeStyles } from "@saleor/macaw-ui";
 export const useStyles = makeStyles(
   theme => ({
     card: {
-      padding: "1px"
+      padding: "1px",
+      borderColor: theme.palette.grey[200],
```

Comment:
```suggestion
      borderColor: theme.palette.divider,
```

or `theme.palette.saleor.main[4]` or whatever the color number is in figma